frontend/points-hack-april20/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:12
+
+WORKDIR /app
+
+COPY package.json .
+COPY yarn.lock .
+COPY lib.js .
+
+RUN yarn install
+
+CMD [ "node", "lib.js" ]
```

Comment:
This probably needs to wait for the localhost server to be running, see https://github.com/CodaProtocol/coda/blob/develop/frontend/bot/Dockerfile for an example